libraries/SPISlave/src/hspi_slave.c

```diff
@@ -98,6 +98,23 @@ void hspi_slave_begin(uint8_t status_len, void * arg)
     ETS_SPI_INTR_ENABLE();
 }
 
+void hspi_slave_end()
+{
+  ETS_SPI_INTR_DISABLE();
+  ETS_SPI_INTR_ATTACH(_hspi_slave_isr_handler, NULL);
```

Comment:
Shouldn't this be something like:
`ETS_SPI_INTR_ATTACH(nullptr, nullptr);`
?

Reply:
sorry, the first argument should be NULL. I change it.

it is a C source code and nullptr is not declared

Reply:
Ah yes, C file, so NULL it is.